### DISCUSSION TITLE
InvisiblePatch works both ways?

### DISCUSSION THREAD

**Discussion by Dexteriority** [winstxnhdw/lc-hax#424] [Q&A] [answered]
They cannot see me but they also disappear from my screen.

**Comment by winstxnhdw** [comment]
![image](https://github.com/winstxnhdw/lc-hax/assets/56998716/33987879-155a-4536-8025-169616b162b9)

Seems fine to me?

**Reply by Dexteriority** [reply]
Maybe its because im a host 🤔
Or a some mod affects it? didn't think they would be strong enough

**Reply by winstxnhdw** [reply]
Why are you using `invis` if you are host anyways 😂

**Reply by winstxnhdw** [reply, marked as answer]
Anyways, I might've fixed it for host. Please help me test in latest.

**Reply by Dexteriority** [reply]
On it

**Reply by Dexteriority** [reply]
Tried it as Host. It works perfectly fine now, thank you!

**Comment by Dexteriority** [comment]
> Why are you using `invis` if you are host anyways 😂

Just testing out some commandsXD
